component/distributed_cache/distributed_cache.go

```diff
@@ -865,7 +865,61 @@ func (dc *DistributedCache) CloseFile(options internal.CloseFileOptions) error {
 }
 
 func (dc *DistributedCache) DeleteFile(options internal.DeleteFileOptions) error {
-	return syscall.ENOTSUP
+	log.Debug("DistributedCache::DeleteFile: Delete file: %s", options.Name)
+	var dcacheErr, azureErr error
+
+	isAzurePath, isDcachePath, isDebugPath, rawPath := getFS(options.Name)
+
+	if isDcachePath {
+		log.Debug("DistributedCache::DeleteFile: Delete for Dcache file: %s", options.Name)
+		err := fm.DeleteDcacheFile(rawPath)
+		if err != nil {
+			log.Err("DistributedCache::DeleteFile: Delete failed for Dcache file: %s:%v", options.Name, err)
```

Comment:
log.Err("DistributedCache::DeleteFile: Delete failed for Dcache file %s: %v", options.Name, err)